packages/frontend/components/Sidebar/Sidebar.tsx

```diff
@@ -247,12 +253,19 @@ const Sidebar: React.FC<SidebarProps> = memo(
 
 interface NoMajorSidebarProps {
   selectedPlan: PlanModel<string>;
+  transferCourses: ScheduleCourse2<unknown>[];
 }
 
 export const NoMajorSidebar: React.FC<NoMajorSidebarProps> = ({
   selectedPlan,
+  transferCourses,
 }) => {
-  const creditsTaken = totalCreditsInSchedule(selectedPlan.schedule);
+  const transferCredits = transferCourses.reduce(
+    (sum, course) => course.numCreditsMin + sum,
+    0
+  );
+  const creditsTaken =
```

Comment:
can we move this logic calculating schedule + transfer courses into one function that returns a credit value?